### DISCUSSION TITLE
threshold based clustering

### DISCUSSION THREAD

**Discussion by maiiabocharova** [facebookresearch/faiss#2630] [Q&A]
First of all thank you a lot for the amazing library!

I want to detect communities in my dataset using Faiss. I know there is a `Kmeans` clustering and there is a `range_search ` functionality. 

Is it possible to make faiss detect communities with centroids and points which are not further than some threshold to their centroid (and if points do not have neighbours just leave them alone)?

**Comment by TheoCGaming** [comment]
[i] This post was used for a mass-mention to advertise a "private videos" website. It appears the offending comment was removed, but beware of more that may flood your inbox.
